common/src/test/java/com/linecorp/centraldogma/common/PathPatternBuilderTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.common;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class PathPatternBuilderTest {
+
+    @Test
+    void testSingleOption() {
+        assertThat(PathPattern.builder().startsWith("/foo/bar").build()
+                              .patternString()).isEqualTo("/foo/bar/**");
+        assertThat(PathPattern.builder().endsWith("json").build()
+                              .patternString()).isEqualTo("/**/json");
```

Comment:
Many users refer to the test cases to see the usage.
It would be helpful to add more realistic inputs.
```suggestion
        assertThat(PathPattern.builder().endsWith("foo.json").build()
                              .patternString()).isEqualTo("/**/foo.json");
```